src/drivers/github.js

```diff
@@ -32,6 +33,13 @@ class Github {
 
     this.repo = repo;
     this.token = token;
+    this.octokit_options = {};
+
+    if (!repo.includes('github.com')) {
```

Comment:
I think that it's not needed maybe? It should work also for github.com so there is not need to do this "special case"?

Reply:
I think we need to logic, because for non-enterprise repos I believe you want to use `api.github.com` as the `baseUrl`.